vault/datadog_checks/vault/metrics.py

```diff
@@ -115,56 +115,60 @@
     'vault_wal_loadWAL': 'vault.wal.loadWAL',
     'vault_wal_persistwals': 'vault.wal.persistwals',
     'vault_wal_flushready': 'vault.wal.flushready',
-    'logshipper_streamWALs_missing_guard': 'logshipper.streamWALs.missing_guard',
```

Comment:
Did vault change the name of these metrics? Just wondering why `vault_` is added to these metrics specifically.

Reply:
> Metrics were incorrectly documented on vault's website. All exposed metrics start with the vault_ prefix

Just added that line to the PR description